tests/Agent/UnitTests/Core.UnitTest/Transformers/TransactionTransformer/ErrorEventMakerTests.cs

```diff
@@ -38,6 +38,7 @@ public class ErrorEventMakerTests
         private IAttributeDefinitionService _attribDefSvc;
         private IAttributeDefinitions _attribDefs => _attribDefSvc?.AttributeDefs;
         private Func<Exception, string> _errorGroupCallback;
+        private const string _expectedErrorGroupAttributeName = "error.group.name";
```

Comment:
Rather than a hard-coded string here, you could use `_attribDefs.ErrorGroup.Name`

Reply:
Yeah, that's something I've gone back and forth on. On the one hand, by using the name from the attribute definition, you only have to change one location. On the other hand, if you make a mistake with that name, the tests won't catch it.

Reply:
Hmm. That might be a good reason for leaving it hard-coded here. I've gone the opposite route in my user id tests, however -- maybe I should change my code before completing that PR.

Reply:
I think that you'll find examples of both approaches in this repo. For some things it's less of a risk than others.